packages/mobile/src/services/offline-downloader/workers/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { reachabilitySelectors } from '@audius/common'
+import queue from 'react-native-job-queue'
+
+import { store } from 'app/store'
+
+const getIsReachable = reachabilitySelectors.getIsReachable
+
+export const startQueueIfOnline = async () => {
```

Comment:
can make this a takeLatest saga, thoughts?

Reply:
yeah good q, what do we think? it seems in some cases we want the queue to start after we stopped it to do some queue removal methods, and other times we probably want to ensure it starts/stops due to reachability changes?

Reply:
Yes, exactly.

I do think we need to be careful to respect the temporary queue.stop() ... queue.start() pattern we use while doing cancel operations